### PR TITLE
some small fixes

### PR DESCRIPTION
**Pull request by misternebula**

### CODE REVIEW

File: QSB/PlayerInfo.cs [hunk 1]
```diff
@@ -53,7 +54,8 @@ public void UpdateStateObjects()
             Translator?.ChangeEquipState(FlagsHelper.IsSet(State, State.Translator));
             ProbeLauncher?.ChangeEquipState(FlagsHelper.IsSet(State, State.ProbeLauncher));
             Signalscope?.ChangeEquipState(FlagsHelper.IsSet(State, State.Signalscope));
-            PlayerRegistry.GetSyncObject<AnimationSync>(NetId)?.SetSuitState(FlagsHelper.IsSet(State, State.Suit));
+            QSB.Helper.Events.Unity.RunWhen(() => PlayerRegistry.GetSyncObject<AnimationSync>(NetId) != null,
```
Comment (misternebula) [inline QSB/PlayerInfo.cs:57]: Fixes some issues where the suit would not appear

File: QSB/PlayerInfo.cs [hunk 2]
```diff
@@ -53,7 +54,8 @@ public void UpdateStateObjects()
...
+                () => PlayerRegistry.GetSyncObject<AnimationSync>(NetId).SetSuitState(FlagsHelper.IsSet(State, State.Suit)));
```
Comment (misternebula) [inline QSB/PlayerInfo.cs:58]: fixes some issues where the suit wouldn't appear

File: QSB/TransformSync/PlayerHUDMarker.cs [hunk 3]
```diff
@@ -38,7 +38,7 @@ private void Update()
             {
                 return;
             }
-            _markerLabel = _player.Name;
+            _markerLabel = _player.Name.ToUpper();
```
Comment (misternebula) [inline QSB/TransformSync/PlayerHUDMarker.cs:41]: Makes hud stuff appear in upper case - more like the game

File: QSB/Utility/DebugLog.cs [hunk 4]
```diff
@@ -18,7 +18,7 @@ public static void ToHud(string message)
             {
                 return;
             }
-            var data = new NotificationData(NotificationTarget.Player, message);
+            var data = new NotificationData(NotificationTarget.Player, message.ToUpper());
```
Comment (misternebula) [inline QSB/Utility/DebugLog.cs:21]: Makes hud stuff appear in upper case - more like the game

**Review by amazingalek — APPROVED** [review]
👍

**Pull request merged**